packages/adafruit-pitft-helper_TEMPLATE/debian/files

```diff
@@ -0,0 +1 @@
+adafruit-pitft-helper_0.0.1-1_armhf.deb utils extra
```

Comment:
i think i also read somewhere that this file shouldn't be in version control, but i can't find the source of that info after searching for 15 mins.